home-dev/README.md

```diff
@@ -21,6 +21,33 @@ How to do it with IDEA and Maven, check the [Maven](#Maven).
 
 
 ##Docker
+**Environment variables**
```

Comment:
I'm not sure that we need it because we already have docker-compose files so why do not use them instead of console commands? Probably we have to choose one option to not confuse new people who read our documentation(to use docker compose or already rewrite documentation for using docker via terminal or console.

Reply:
We publish our application to dockerhub as public images so we need to provide some information on how to use them and which configurations are available.